### PR TITLE
improve usability when mix start with no loaded project.

### PR DESCRIPTION
**Pull request by yann300**
https://github.com/ethereum/webthree-umbrella/issues/105

### CODE REVIEW

File: qml/Watchers.qml [hunk 1]
```diff
@@ -102,7 +107,7 @@ Column {
 		}
 	}
 
-	RowLayout
+		 RowLayout
```
Comment (LianaHus) [inline qml/Watchers.qml]: wrong indent :)

**Comment by winsvega** [comment]
so now it is default project always loaded when starting mix?

Menu sections Scenario and Debug still contains active elements. 
Though now this elements not responding when no project loaded it is still active elements. (should be disabled)

**Comment by yann300** [comment]
I've done only a few things from your comments... @LianaHus is taking care of the rest.

File: qml/StateDialog.qml [hunk 2]
```diff
@@ -321,6 +323,7 @@ Dialog {
 										{
 											stateAccounts.splice(styleData.row, 1)
```
Comment (LianaHus) [inline qml/StateDialog.qml:324]: we should also prevent delete of last account

File: qml/LogsPane.qml [hunk 3]
```diff
@@ -298,6 +303,14 @@ Rectangle
 			spacing: logStyle.generic.layout.headerButtonSpacing
 			height: parent.height
 
+			DefaultLabel
+			{
+				id: labelFilter
+				text: qsTr("Filter:")
+				verticalAlignment: Text.AlignVCenter
+				width: 50
+			}
```
Comment (LianaHus) [inline qml/LogsPane.qml]: "Filter" label looks like another button the user can click. We could add more info to tooltip. and I think we should make the background color a little bit more darker for toggled state

File: qml/StatusPane.qml [hunk 4]
```diff
@@ -60,19 +53,31 @@ Rectangle {
...
+		status.state = state
+		status.text = text
+		status.font.italic = false
+	}
+
 	function clear()
 	{
 		status.state = "";
-		status.text = "";
+		status.text = "Empty";
```
Comment (LianaHus) [inline qml/StatusPane.qml]: I can also suggest <no logs>

**Pull request merged**